<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlashcardEditor</class>
 <widget class="QDialog" name="FlashcardEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btnAddImage {
color: white;
background: #4CAF50;
border-radius: 0px;
}

QPushButton:hover#btnAddImage {
background: #43A047
}

QPushButton:disabled#btnAddImage {
background-color: rgb(204, 204, 204);
}</string>
  </property>
  <widget class="QPushButton" name="btnAddImage">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Insert image</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtFront">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>560</width>
     <height>140</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblFrontText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Front:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>640</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDecks">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtBack">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>560</width>
     <height>140</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDeck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DECK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBackText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Back:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbDecks</tabstop>
  <tabstop>txtFront</tabstop>
  <tabstop>txtBack</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
